src/DependencyInjection/SonataUserExtension.php

```diff
@@ -178,7 +178,7 @@ public function configureGoogleAuthenticator($config, ContainerBuilder $containe
         }
 
         $container->setParameter('sonata.user.google.authenticator.forced_for_role', $config['google_authenticator']['forced_for_role']);
-        $container->setParameter('sonata.user.google.authenticator.ip_white_list', $config['google_authenticator']['ip_white_list']);
+        $container->setParameter('sonata.user.google.authenticator.trusted_ip_list', $config['google_authenticator']['trusted_ip_list']);
```

Comment:
This would be BC-break, people can still use the `ip_white_list` option, so here we should also check if `ip_white_list` exists and use it. A test verifying that this behavior still works would be nice.

Reply:
@franmomu I've marked `ip_white_list` as a deprecated parameter, is this enough?

Reply:
`ip_white_list` is a configuration node. A container parameter is something else entirely. AFAIK You cannot deprecate it, but maybe I'm wrong. IMO both parameters from whatever configuration node is defined in the config (after validating that one and only one of them is defined as I described above).

Reply:
What I meant is removing `sonata.user.google.authenticator.ip_white_list` from the container is BC-break because someone can be using that parameter (it's unlikely, but you never know), if there is no way to deprecate a parameter in the container, an upgrade note I think it's enough and we will remove it on the next major version.